extensions/wikia/PotentialMemberPageExperiments/templates/PMPEntryPoint.mustache

```diff
@@ -1,4 +1,4 @@
-<div class="pmp-entry-point pmp-entry-point-{{bannerType}} clearfix">
+<div class="pmp-entry-point pmp-entry-point-{{bannerType}}">
```

Comment:
This will now output the second class as `pmp-entry-point-` since `bannerType` isn't set. Though banner type will probably be useful for the third entry point variation.